packages/axelar-wasm-std/src/voting.rs

```diff
@@ -137,6 +140,8 @@ pub trait Poll {
         Self: Sized;
     // returns the cumulated poll result
     fn result(&self) -> PollResult;
```

Comment:
should PollResult be an associated type? It doesn't really work well for WorkerSet polls. Also, I don't understand the field `consensus_participants`, shouldn't we have a list per message instead of one for the whole poll?